### ISSUE TITLE
GCD

### ISSUE THREAD

**Issue by lackthebest** [Smolderforge/issues#460]
Hello, I have a question, I played bc on the official server, on many pirates, but I have never seen GCD, the paladin should not have it, + gcd is triggered by racial ability, which should not be, more than 10 years in wow I see this 1 time. The paladin should not have gcd on cleanse, on holy shock / holy light and blessing abilities, once again on the official server, the paladin did not have gcd for these abilities

**Comment by henhouse** [comment]
I will need proof of this. I played a Paladin during TBC and I definitely remember GCD on Cleanse, but perhaps my memory fails me.

**Comment by lackthebest** [comment]
Unfortunately, it is very difficult to find a proof, as it is practically impossible to find a video from the official tbc server, but I remember exactly when playing on the classics and tbc that the paladin does not have a GCD on cleanse, since he played for this class since 2004, GCD was only on the rest of the holy spell, and GCD should not be on the racial ability, it has nothing to do with holy, while there was a subscription to the classic played as a paladin, there is no such thing

**Comment by lackthebest** [comment]
Also, seal of blood does not work correctly after using jugement, the damage returned to the paladin should not be instantaneous, but with a delay if I'm not mistaken 0.5 sec

**Comment by Naksgh** [comment]
If you're implying that cleanse had no GCD at all, similar to Counterspell, then its hilariously wrong. Of course there was GCD on Cleanse, else you could just spam a hundred of it per global... or even more. Makes no sense to suggest such thing, although it would make palas significantly more viable :)

**Comment by lackthebest** [comment]
Yes, here is my mistake on cleanse, after watching videos from the classics and old tbc videos of gcd, of course there is something that creates discomfort for me and why it is difficult to play on this server, I have a delay of about 1 sec, although the ping is 20-40, such an impression that it might be lags on the server for everyone, but I could be wrong, because of this it seems to me that gcd is almost an eternity. But the racial ability should not call GCD holy spells, different types of mechanics, like the damage from seal of blood should not be instant

**Comment by henhouse** [comment]
Can you link some of these videos from TBC that show this behavior with GCD?
